pkg/ingress/tls.go

```diff
@@ -110,6 +111,10 @@ func (i *Tls) Process() error {
 }
 
 func (i *Tls) RequestCert() error {
+	// sanity check
+	if i.IngressTLS.SecretName == "" {
+		return errors.New("ingress has an empty SecretName. Must fail")
```

Comment:
Could we change this to `Ingress has an empty secretName. Skipping certificate retrieval`?